<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentedit</class>
 <widget class="QDialog" name="studentedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑学生信息</string>
  </property>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请确认信息填写正确</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>158</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="quit_button">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="adress">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>112</y>
     <width>404</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephone_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>179</y>
     <width>404</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="gender_combo">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>141</y>
     <width>410</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>跨性别</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>81</y>
     <width>404</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>81</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>179</y>
     <width>27</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>143</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>210</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>电子邮件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>112</y>
     <width>27</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>404</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编辑信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
